ts/src/pro/bitget.ts

```diff
@@ -595,6 +595,7 @@ export default class bitget extends bitgetRest {
             marketId = market['id'];
             messageHash = messageHash + ':' + symbol;
         }
+        const isStop = this.safeValue (params, 'stop', false);
```

Comment:
@dmunozfer also, next time we add a parameter like this we have to omit it from `params`